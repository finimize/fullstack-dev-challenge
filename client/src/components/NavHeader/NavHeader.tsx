import React from 'react'
import { Box, Image } from '@chakra-ui/react'
import images from '../../images'

const NavHeader = () => (
    <Box
        display="flex"
        flexDirection="row"
        justifyContent="space-between"
        px={6}
        py={4}
        bg="blue100"
    >
        <Image src={images.fullBrandLogo} alt="Finimize" width="160px" />
    </Box>
)

export default NavHeader
