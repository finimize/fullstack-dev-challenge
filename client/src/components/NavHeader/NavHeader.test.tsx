import React from 'react'
import { screen } from '@testing-library/react'
import { renderForTest } from '../../test'

import NavHeader from './NavHeader'

describe('NavHeader', () => {
    it('renders successfully and Logo is visible', () => {
        renderForTest(<NavHeader />)

        const logo = screen.getByRole('img')
        expect(logo).toHaveAttribute('alt', 'Finimize')
    })
})
