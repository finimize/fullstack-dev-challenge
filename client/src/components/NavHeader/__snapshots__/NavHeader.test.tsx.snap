// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders learn react link 1`] = `
<div>
  <div
    class="css-1c4hsg8"
  >
    <img
      alt="Finimize"
      class="chakra-image__placeholder css-u4ybgy"
    />
  </div>
</div>
`;
